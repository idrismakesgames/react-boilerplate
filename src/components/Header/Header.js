import React from 'react';
import classes from './Header.module.scss';

export default function Header() {
  return (
    <div className={classes.header}>
      <h1>Welcome to my `Boilerplate Test Project`</h1>
    </div>
  );
}
