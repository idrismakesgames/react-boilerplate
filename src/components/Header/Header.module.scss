.header {
  background-color: #fff;
  text-align: center;
  margin-bottom: 40px;
}
