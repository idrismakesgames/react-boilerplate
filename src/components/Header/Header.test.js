import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {
  const header = shallow(<Header />);

  it('renders properly', () => {
    expect(header).toMatchSnapshot();
  });

  it('displays footer text', () => {
    expect(header.find('h1').text()).toEqual('Welcome to my `Boilerplate Test Project`');
  });
});
