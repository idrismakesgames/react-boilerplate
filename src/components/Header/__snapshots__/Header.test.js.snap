// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders properly 1`] = `
<div
  className="header"
>
  <h1>
    Welcome to my \`Boilerplate Test Project\`
  </h1>
</div>
`;
